examples/certificate-check-upload.yaml

```diff
@@ -0,0 +1,40 @@
+# An example Job to run a certificate check of OpenShift's internal
+# certificate status from within OpenShift.
+#
+# It uses the openshift/openshift-ansible container image.
+#
+# It expects the Ansible inventory file to be available from the 'hosts'
+# key of a ConfigMap named 'inventory' and the ssh key in the 'ssh-privatekey'
+# key of a Secret named 'sshkey'
+# (see README_CONTAINER_IMAGE.md in the top level dir for more details).
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: certificate-check
+spec:
+  containers:
+  - name: openshift-ansible
+    image: openshift/openshift-ansible
+    env:
+    - name: PLAYBOOK_FILE
+      value: playbooks/certificate_expiry/easy-mode-upload.yaml
+    - name: INVENTORY_FILE
+      value: /tmp/inventory/hosts       # from configmap vol below
+    - name: ANSIBLE_PRIVATE_KEY_FILE    # from secret vol below
+      value: /opt/app-root/src/.ssh/id_rsa/ssh-privatekey
+    - name: CERT_EXPIRY_WARN_DAYS
+      value: "45"      # must be a string, don't forget the quotes
```

Comment:
The ever curious in me... why must it be a string? Oh my, YAML "magic" types + the way how the library we use parses YAML to fit into a Go string...